scripts/build_snp.sh

```
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+build_kernel_packages() {
+    local build_dir=$1
+    local config=$2
+    pushd ${build_dir}
```

Shellcheck (0.10.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)